### PR TITLE
Fix `extract_values` invert parameter for multi-component values

### PR DESCRIPTION
**Pull request by user27182**
The `invert` parameter of the `extract_values` filter (from #5963 and #6001) does not work as expected for multi-component values. This PR fixes this.

This bug was discovered as part of #5968 , which uses this filter in some tests.

### CODE REVIEW

**Comment by user27182** [comment]
@pyvista-bot preview

File: tests/core/test_dataset_filters.py [hunk 1]
```diff
@@ -2582,12 +2606,18 @@ def test_extract_values_component_mode(
...
     actual_colors = single_mesh['colors']
-    assert np.array_equal(actual_points, expected)
-    assert np.array_equal(actual_colors, expected)
+
+    if invert:
+        assert np.array_equal(actual_points, expected_invert)
+        assert np.array_equal(actual_colors, expected_invert)
+    else:
+        assert np.array_equal(actual_points, expected)
+        assert np.array_equal(actual_colors, expected)
```
Comment (tkoyama010) [inline tests/core/test_dataset_filters.py:2620]:
```suggestion

    assert np.array_equal(actual_points, expected_invert) if invert else np.array_equal(actual_points, expected)
    assert np.array_equal(actual_colors, expected_invert) if invert else np.array_equal(actual_colors, expected)
```

File: tests/core/test_dataset_filters.py [hunk 2]
```diff
@@ -2582,12 +2606,14 @@ def test_extract_values_component_mode(
...
         component_mode=component_mode,
         split=split,
+        invert=invert,
     )
     single_mesh = extracted.combine() if split else extracted
     actual_points = single_mesh.points
     actual_colors = single_mesh['colors']
-    assert np.array_equal(actual_points, expected)
-    assert np.array_equal(actual_colors, expected)
+
```
Comment (tkoyama010) [inline tests/core/test_dataset_filters.py:2614]:
```suggestion
```

**Comment by tkoyama010** [comment]
pre-commit.ci autofix

**Comment by tkoyama010** [comment]
pre-commit.ci autofix

**Review by tkoyama010 — APPROVED** [review]
LGTM

**Pull request merged**